src/v/model/transform.cc

```diff
@@ -102,6 +170,26 @@ std::ostream& operator<<(std::ostream& os, const transform_metadata& meta) {
     return os;
 }
 
+void transform_metadata::serde_write(iobuf& out) const {
+    serde::write(out, name);
+    write(out, input_topic);
```

Comment:
No serde namespace?

Reply:
surprisingly (to me anyway) apparently not. we're shooting for this overload:

https://github.com/redpanda-data/redpanda/blob/fd1434d925ceb5e50a67cd10d17043e5904b509b/src/v/model/metadata.h#L363-L367

perhaps making that a free function in `namespace serde` would be a good cleanup though

Reply:
Ah ADL before we had the tag invoke pattern

Reply:
Just leave it it's fine for now

Reply:
oh yeah wasn't going to change it today